pkg/controller/nginxingresscontroller/utils.go

```diff
@@ -98,6 +101,16 @@ func generatePodArgs(instance *k8sv1alpha1.NginxIngressController) []string {
 		}
 	}
 
+	if instance.Spec.EnableCRDs {
```

Comment:
you have extensive testing of this function in `utils_test.go`. Perhaps it make sense to cover this case as well.